forwarder_test.go

```diff
@@ -222,11 +189,13 @@ func TestUpdateResourceIpAddressAndCertificateInfo(t *testing.T) {
 			assert.NoError(err)
 
 			err = updateResourceIpAddressAndCertificateInfo(testReq, client, resourceURL)
+
 			if tt.expectedStatus != http.StatusOK {
 				assert.Error(err)
 			} else {
 				assert.NoError(err)
 			}
+			assert.Equal(tt.expectedStatus, http.StatusOK, "Expected status OK")
```

Comment:
What is the reason to remove negative test case scenario?
In negative case scenario, this condition should fail. So this should not be present here.